app/spectaql/compose-paths.js

```diff
@@ -161,30 +166,30 @@ module.exports = function composePaths ({ domains, graphQLSchema, jsonSchema })
                     // to make things look "OK"
                     //
                     // eslint-disable-next-line no-unused-vars
-                    const expand = typeof _.get(gqlField, 'type.getFields') === 'function'
-                        ? Object.values(gqlField.type.getFields()).reduce(
-                            (acc, field) => {
-                                const {
-                                    name,
-                                    type,
-                                } = field
-
-                                if (typeof type.getFields === 'function') {
-                                    // Maybe also filter the 'select' by the fields that are in the returnTypeProperties?
-                                    // They will presumably have already respected the undocumentedness.
-                                    //
-                                    // Note to self on the above statement: I think it is already done.
-                                    acc.push({
-                                        field: name,
-                                        select: Object.keys(type.getFields()),
-                                    })
-                                }
-
-                                return acc
-                            },
-                            [],
-                        )
-                        : []
+                    // const expand = typeof _.get(gqlField, 'type.getFields') === 'function'
+                    //     ? Object.values(gqlField.type.getFields()).reduce(
+                    //         (acc, field) => {
+                    //             const {
+                    //                 name,
+                    //                 type,
+                    //             } = field
+
+                    //             if (typeof type.getFields === 'function') {
+                    //                 // Maybe also filter the 'select' by the fields that are in the returnTypeProperties?
+                    //                 // They will presumably have already respected the undocumentedness.
+                    //                 //
+                    //                 // Note to self on the above statement: I think it is already done.
+                    //                 acc.push({
+                    //                     field: name,
+                    //                     select: Object.keys(type.getFields()),
+                    //                 })
+                    //             }
+
+                    //             return acc
+                    //         },
+                    //         [],
+                    //     )
+                    //     : []
```

Comment:
Just delete it if unnecessary?

Reply:
For now I want to keep it around so I don't forget about this capability/feature...but I will probably remove it in the revamp...